src/layers/legacy/medVtkInria/vtkVisuManagement/vtkTessellatedSphereSource.cxx

```diff
@@ -225,12 +225,21 @@ void TesselateTriangles(vtkPoints* oldVertices,vtkCellArray* oldTriangles,
         const vtkIdType c3 = 3;
         triangles->InsertNextCell(c3,triPointsIds);
         std::swap(triPointsIds[0],triPointsIds[1]);
-        std::swap(pts[0],pts[1]);
 
-        for (int j=0;j<3;++j) {
-            std::swap(triPointsIds[j],pts[j]);
+        // Avoid const*
+        vtkIdType *tmpPts;
+        for (int j=0; j<3; ++j)
+        {
+            tmpPts[j]  = pts[j];
+        }
+        
+        std::swap(tmpPts[0],tmpPts[1]);
+
+        for (int j=0;j<3;++j) 
+        {
+            std::swap(triPointsIds[j],tmpPts[j]);
             triangles->InsertNextCell(c3,triPointsIds);
-            std::swap(triPointsIds[j],pts[j]);
```

Comment:
you should be able to do something like this instead:
`std::swap(triPointsIds[j], const_cast<vtkIdType>(pts[j]));`

Reply:
(also I just realizedthe `tmpPts` array hasn't been allocated anyways)

Reply:
Solved, i used a std::copy method. The const cast is not needed here we can use a regular way

Reply:
The const cast was to avoid having to create a temporary array.